.github/workflows/ci-publish-release.yml

```diff
@@ -24,13 +24,14 @@ jobs:
         uses: battila7/get-version-action@v2
 
       - name: Checkout code
-        uses: actions/checkout@v2
+        uses: actions/checkout@v3 
```

Comment:
Updated to more recent versions of these actions as they were throwing deprecated warnings